docs/rules/no-array-for-each.md

```diff
@@ -1,6 +1,10 @@
 # Prefer `for…of` over `Array#forEach(…)`
 
-A [`for…of` statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) is more readable than [`Array#forEach(…)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach).
+Benefits of [`for…of` statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) over [`Array#forEach(…)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) can include:
+
+- Better readability
+- No additional scope (easier debugging)
```

Comment:
This is not entirely correct. It does have a scope, just a block scope instead of a function scope.

Reply:
Good point, removed this line.